axelor-account/src/main/java/com/axelor/apps/account/service/InvoiceVisibilityServiceImpl.java

```diff
@@ -112,6 +112,7 @@ public boolean getManagePfpCondition(Invoice invoice) {
   public boolean getOperationTypePurchaseCondition(Invoice invoice) {
     return invoice.getOperationTypeSelect() == InvoiceRepository.OPERATION_TYPE_SUPPLIER_PURCHASE
         || (invoice.getOperationTypeSelect() == InvoiceRepository.OPERATION_TYPE_SUPPLIER_REFUND
+            && invoice.getCompany() != null
             && invoice.getCompany().getAccountConfig().getIsManagePFPInRefund());
```

Comment:
Please improve this code and use AccountConfigService.getAccountConfig(Company company) instead of invoice.getCompany().getAccountConfig(). 
If accountConfig is null, then we still have an NPE